appyx-interactions/android/src/main/kotlin/com/bumble/appyx/interactions/sample/TestDriveExperiment.kt

```diff
@@ -36,23 +36,20 @@ import com.bumble.appyx.interactions.core.model.transition.Operation.Mode.IMMEDI
 import com.bumble.appyx.interactions.core.model.transition.Operation.Mode.KEYFRAME
 import com.bumble.appyx.interactions.core.model.transition.Update
 import com.bumble.appyx.interactions.core.ui.helper.InteractionModelSetup
-import com.bumble.appyx.interactions.sample.NavTarget.Child1
-import com.bumble.appyx.interactions.theme.appyx_dark
 import com.bumble.appyx.transitionmodel.testdrive.TestDrive
 import com.bumble.appyx.transitionmodel.testdrive.TestDriveModel
+import com.bumble.appyx.transitionmodel.testdrive.operation.next
 import com.bumble.appyx.transitionmodel.testdrive.ui.TestDriveMotionController
 import com.bumble.appyx.transitionmodel.testdrive.ui.TestDriveMotionController.Companion.toTargetUiState
-import com.bumble.appyx.transitionmodel.testdrive.operation.next
 
 
-@ExperimentalMaterialApi
 @Composable
 fun TestDriveExperiment() {
     val coroutineScope = rememberCoroutineScope()
 
     val model = remember {
         TestDriveModel(
-            element = Child1,
+            element = NavTarget.Child1,
```

Comment:
Actually, `NavTarget` as a name shouldn't be used when in the context of interactions (and nothing to do with navigation), can you please update to `InteractionTarget`